### ISSUE TITLE
[SUGGESTION] Recipes Configs

### ISSUE THREAD

**Issue by Neodark7** [McJty/RFTools#344]
Can you add "other mods recipes configs" ?
By that i mean adding a config to choose what recipes model to use between Thermal Expansion and EnderIO for example, like MFR and SimplyJetpacks does.
idk if i explained my suggestion clearly ^^

**Comment by McJty** [comment]
I'd like to work for this (probably not for next version but for the version after that) but do you have a proposal on how you would want to see this done?

**Comment by Neodark7** [comment]
Yeah i defintely can make screenshots of all the modified recipes for you or MT scripts as you want i've already started for EnderIO recipes. My goal here is to make harder recipes but what's yours ? Do you plan to make it by default if EnderIO is installed or make a "Hardmode recipes" config ? Cause my recipes are not hardcore but pretty difficult.

**Comment by McJty** [comment]
I'm not sure yet. Just checking options. I wasn't specifically planning to look for EnderIO and/or Thermal Expansion but instead look if certain materials were present like tin, lead, silver, ... and then use those where appropriate. By default it shouldn't necessarily be harder then it is now but a bit better integrated with the other tech mods but hard recipe option can certainly be considered

**Comment by Neodark7** [comment]
Hi ! sorry for the delay, i was a bit away from minecraft this month, here some ideas for some machines https://gist.github.com/Neodark7/cfa43668e50115af1e27 with " "Hard" EIO Themed Recipes".

The goal here (for my modpack) is to make your mod only accessible after the "Steel Discovery" and the "EIO Tier" using advanced eio alloy/components, i haven't done all the machines ofc cause if you don't like it, it would have been wasted time ^^ but if you like it / plan to add it as a "EIO integration config" i can do alternative recipes for everything in your mod :)

if you want to test this in game i made it with the latest versions of everything ( Enderio+core & RFTools+core & modloader )
